scalafix/src/main/scala/org/scalajs/dom/scalafix/MutableState.scala

```diff
@@ -47,10 +52,20 @@ final class MutableState {
 
     val b = SortedSet.newBuilder[Result]
 
+    def deprecationSuffix(ver: Option[String]): String =
+      ver match {
+        case None    => ""
+        case Some(v) => s"  (@deprecated in $v)"
```

Comment:
```suggestion
        case Some(v) => s" (@deprecated in $v)"
```
Unless it was intentional? :)

Reply:
Yeah, double-space is intentional cos when I first tried single space it was a bit too cluttered for easy reading. This is already on the `series/v1.x` branch btw, here I'm just showing additional deprecations without changing the format.